### PR TITLE
React function components

### PR DESCRIPTION
**Pull request by OSAdams**
image

### CODE REVIEW

File: react-function-components/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+class CustomButton extends React.Component {
+  render() {
+    return <button>Click me!</button>;
+  }
+}
```
Comment (scbowler) [inline react-function-components/src/index.jsx]: This should be a `function` component

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but you should be using a `function` component, not a `class` component. Let me know if you have any questions.

**Comment by OSAdams** [comment]
image

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**